packages/kbn-securitysolution-io-ts-alerting-types/src/throttle/index.ts

```diff
@@ -8,7 +8,12 @@
 
 import * as t from 'io-ts';
 
-export const throttle = t.string;
+export const throttle = t.union([
+  t.null,
+  t.literal('onActiveAlert'),
+  t.literal('onThrottleInterval'),
+  t.literal('onActionGroupChange'),
+]);
```

Comment:
Not sure here about `onActionGroupChange`, as I haven't seen/understood yet it usage, but looks it should be part of the type def